libraries/ssh_config_helpers.rb

```diff
@@ -19,7 +19,7 @@ def parse_file(path)
         IO.foreach(path) do |line|
           next if line.match(/^\s*(#|\r?\n|\s*$)/) # skip lines with only comments or whitespace
 
-          matchdata = line.match(/^\s*([h|H]ost)(.*$)/)
+          matchdata = line.match(/^\s*([h|H]ost\s)(.*$)/)
```

Comment:
can you add a `+` after the `\s`?  That will ensure that we FORCE at least one whitespace.  After that, I'm happy to accept it and cut a release.